torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2702,6 +2703,11 @@ def create_symbolic_sizes_strides_storage_offset(
         # The order of checking the guards matters. In this specific example:
         # If True branch guard check precedes False branch and for True branch, y.size(0) check precedes x == True,
         # we may have an unnessary shape speciliazation for y.
+        if ex.is_nested:
+            # For nested tensor subclasses, we rely on transform subclass logic
+            # to create symbolic sizes/strides/storage_offset.
+            return None, None, None
```

Comment:
It would seem better to just not call this function at all in this case (and assert `not ex.is_nested`)